src/ResponseMerger/NuxtResponseMerger.php

```diff
@@ -29,8 +29,8 @@ public function mergeResponses(ResponseInterface $backendResponse, ResponseInter
 
     // Finally, merge responses and serve them.
     $page = $frontendResponse->getBody()->__toString();
-    $page = preg_replace('/<title(.*)>*<\/title>/', '<title$1>' . strip_tags($data['title']) . '</title>', $page);
-    $page = preg_replace('/<main role="main"(.*)><\/main>/', '<main role="main"$1>' . $data['content'] . '</main>', $page);
+    $page = preg_replace('/<title((?:\s*[a-zA-Z\-0-9]+(?:="[^"]*")*\s*)*)>[^<]*<\/title>/', '<title$1>' . strip_tags($data['title']) . '</title>', $page);
```

Comment:
please make this more clear what this goes for. e.g. add a class constant ATTRIBUTE_REGEX for the first part